<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileTransferWindow</class>
 <widget class="QWidget" name="FileTransferWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件传输工具</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget {
    background-color: #2a2a2a;
    color: #e0e0e0;
    font-family: Arial;
}
QTableWidget {
    background-color: #333333;
    gridline-color: #444444;
    border: 1px solid #444444;
}
QTableWidget::item:selected {
    background-color: #444444;   /* 选中项灰色 */
    color: #ffffff;
}
QTableWidget::item:selected:active {
    background-color: #287bff;   /* 有焦点时蓝色 */
    color: #ffffff;
}
QTableWidget::item:selected:!active {
    background-color: #444444;   /* 失去焦点时灰色 */
    color: #ffffff;
}
QHeaderView::section {
    background-color: #3a3a3a;
    color: #e0e0e0;
    padding: 4px;
    border: none;
}
QComboBox {
    background-color: #2a2a2a;
    color: #e0e0e0;
    padding: 4px;
    border: 1px solid #444444;
}
QPushButton {
    background-color: #505050;
    color: #e0e0e0;
    border: 1px solid #444444;
    padding: 5px 10px;
    min-width: 80px;
}
QPushButton#uploadButton, QPushButton#downloadButton {
    background-color: #287bff;
    color: white;
    font-weight: bold;
    border: 1px solid #1a5dcc;
}
QPushButton#uploadButton:hover, QPushButton#downloadButton:hover {
    background-color: #539dff;
}
QPushButton#uploadButton:pressed, QPushButton#downloadButton:pressed {
    background-color: #1a5dcc;
}
QTableCornerButton::section {
	background: #2a2a2a;  /* 匹配深色背景 */
	border: none;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="localGroupBox">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>本地设备</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="localPathCombo"/>
        </item>
        <item>
         <widget class="QTableWidget" name="localTable">
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="columnCount">
           <number>4</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>30</number>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>100</number>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>30</number>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>30</number>
          </attribute>
          <column>
           <property name="text">
            <string>名称</string>
           </property>
           <property name="textAlignment">
            <set>AlignLeading|AlignVCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>大小</string>
           </property>
           <property name="textAlignment">
            <set>AlignLeading|AlignVCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>修改时间</string>
           </property>
           <property name="textAlignment">
            <set>AlignLeading|AlignVCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>类型</string>
           </property>
           <property name="textAlignment">
            <set>AlignLeading|AlignVCenter</set>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="uploadButton">
          <property name="text">
           <string>上传</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="remoteGroupBox">
       <property name="title">
        <string>远程设备</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QComboBox" name="remotePathCombo"/>
        </item>
        <item>
         <widget class="QTableWidget" name="remoteTable">
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="columnCount">
           <number>4</number>
          </property>
          <column>
           <property name="text">
            <string>名称</string>
           </property>
           <property name="textAlignment">
            <set>AlignLeading|AlignVCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>大小</string>
           </property>
           <property name="textAlignment">
            <set>AlignLeading|AlignVCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>修改时间</string>
           </property>
           <property name="textAlignment">
            <set>AlignLeading|AlignVCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>类型</string>
           </property>
           <property name="textAlignment">
            <set>AlignLeading|AlignVCenter</set>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="downloadButton">
          <property name="text">
           <string>下载</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="line">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="transferLogLabel">
     <property name="styleSheet">
      <string notr="true">font-weight: bold; font-size: 12pt;</string>
     </property>
     <property name="text">
      <string>传输记录</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="transferLogTable">
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <column>
      <property name="text">
       <string>名称</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>状态</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>操作</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
